### PR TITLE
adb reboot: add page

### PR DESCRIPTION
**Pull request by DarkMatter-999**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).

### CODE REVIEW

**Comment by DarkMatter-999** [comment]
added support for `adb reboot` command.

#13997

File: pages/common/adb-reboot.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# adb reboot
+
+> Android Debug Bridge Install: reboot an Android emulator instance or connected Android devices.
+> More information: <https://developer.android.com/tools/adb>.
```
Comment (Managor) [inline pages/common/adb-reboot.md:4]: This page doesn't seem to contain any instructions on how to use adb reboot. Lets use manned instead.
```suggestion
> More information: <https://manned.org/adb>.
```

File: pages/common/adb-reboot.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+`adb reboot bootloader`
+
+- Reboot the device into recovery mode:
+
+`adb reboot recovery`
+
+- Reboot the device into fastboot mode:
+
+`adb reboot fastboot`
```
Comment (Managor) [inline pages/common/adb-reboot.md:20]: Now that I look at the manpage, does the fastboot subcommand even exist or do anything?

Comment (Managor) [reply pages/common/adb-reboot.md:20]: I tested it. Despite not being listed in the man page, it does indeed exist.

File: pages/common/adb-reboot.md (hunk 1 above)
Comment (spageektti) [reply pages/common/adb-reboot.md:4]: It needs to be updated on other pages then.
https://github.com/tldr-pages/tldr/blob/main/pages/common/adb-shell.md

Comment (DarkMatter-999) [reply pages/common/adb-reboot.md:4]: yeah sure

**Review by Managor — APPROVED** [review]

File: pages/common/adb-reboot.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# adb reboot
+
+> Android Debug Bridge Install: reboot an Android emulator instance or connected Android devices.
```
Comment (Managor) [inline pages/common/adb-reboot.md:3]: This should be as brief as possible
```suggestion
> Reboot a connected Android device or emulator.
```

Comment (DarkMatter-999) [reply pages/common/adb-reboot.md:3]: TBH this was taken from `pages/common/adb-shell.md`.

but sure.

Comment (Managor) [reply pages/common/adb-reboot.md]: I see, thanks. I'll make that one briefer too.

**Review by gutjuri — APPROVED** [review]
Thanks!

**Review by spageektti — APPROVED** [review]

**Pull request merged**